templates/stylelint.config.cjs

```diff
@@ -1,40 +1,6 @@
 /** @type { import('stylelint').Config } */
 const config = {
-  extends: ['stylelint-config-recommended'],
-  rules: {
-    'no-descending-specificity': null,
-    // Allow files without any styles
-    'no-empty-source': null,
-  },
-  overrides: [
-    {
-      files: ['**/*.css', '**/*.scss', '**/*.sass', '**/*.less'],
-      extends: ['stylelint-config-recommended-scss'],
-      rules: {
-        'scss/at-rule-no-unknown': [
-          true,
-          {
-            ignoreAtRules: ['tailwind'],
-          },
-        ],
-      },
-    },
-    {
-      files: ['**/*.module.css', '**/*.module.scss', '**/*.module.sass'],
-      extends: ['stylelint-config-css-modules'],
-    },
-    {
-      files: [
-        '**/*.js',
-        '**/*.cjs',
-        '**/*.mjs',
-        '**/*.jsx',
-        '**/*.ts',
-        '**/*.tsx',
-      ],
-      customSyntax: 'postcss-styled-syntax',
-    },
-  ],
```

Comment:
simplify config similar to https://github.com/upleveled/eslint-config-upleveled/pull/220/files#r1166967732